### ISSUE TITLE
Instance creation issue(Scaleway)

### ISSUE THREAD

**Issue by c29r3** [trailofbits/algo#1502]
The error occurs after specifying the api key (log below). The problem occurs only if i'm changing the parameter "size" in the file "config.cfg":
DEFAULT

```
  scaleway:
    size: START1-S
    image: Ubuntu Bionic Beaver
    arch: x86_64
```

EDITED

```
  scaleway:
    size: DEV1-S
    image: Ubuntu Bionic Beaver
    arch: x86_64
```
DEV1-S is available and can be created manually

**LOG**

```
TASK [cloud-scaleway : Set scaleway facts] **************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [cloud-scaleway : Gather Scaleway organizations facts] *********************************************************************************************************************************************************************************
ok: [localhost]

TASK [cloud-scaleway : Get images] **********************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [cloud-scaleway : Set cloud specific facts] ********************************************************************************************************************************************************************************************
ok: [localhost]

TASK [cloud-scaleway : Create a server] *****************************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"msg": "The conditional check 'algo_instance.msg.public_ip' failed. The error was: error while evaluating conditional (algo_instance.msg.public_ip): 'ansible.utils.unsafe_proxy.AnsibleUnsafeText object' has no attribute 'public_ip'"}

TASK [include_tasks] ************************************************************************************************************************************************************************************************************************
included: /home/ss/algo-master/playbooks/rescue.yml for localhost

TASK [debug] ********************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "fail_hint": [
        "Sorry, but something went wrong!",
        "Please check the troubleshooting guide.",
        "https://trailofbits.github.io/algo/troubleshooting.html"
    ]
}
```

**Comment by TC1977** [comment]
Try running it again with `./algo -vvv`. It doesn't sound like an actual problem with the Algo script (if it were then Ansible would have complained about `cloud_providers.scaleway.size` or `commercial_type`), more like a timeout while creating the Scaleway instance. Lots of issues concerning timeouts with Scaleway in the past.

**Comment by c29r3** [comment]
I solved the problem by adding DEV1-S to the SCALEWAY_COMMERCIAL_TYPES variable in file scaleway_compute.py. Now it's work fine for me

```
SCALEWAY_COMMERCIAL_TYPES = [

    # Virtual ARM64 compute instance
    'ARM64-2GB',
    'ARM64-4GB',
    'ARM64-8GB',
    'ARM64-16GB',
    'ARM64-32GB',
    'ARM64-64GB',
    'ARM64-128GB',

    # Baremetal
    'C1',  # ARM64 (4 cores) - 2GB
    'C2S',  # X86-64 (4 cores) - 8GB
    'C2M',  # X86-64 (8 cores) - 16GB
    'C2L',  # x86-64 (8 cores) - 32 GB

    # Virtual X86-64 compute instance
    'START1-XS',  # Starter X86-64 (1 core) - 1GB - 25 GB NVMe
    'START1-S',  # Starter X86-64 (2 cores) - 2GB - 50 GB NVMe
    'START1-M',  # Starter X86-64 (4 cores) - 4GB - 100 GB NVMe
    'START1-L',  # Starter X86-64 (8 cores) - 8GB - 200 GB NVMe
    'X64-15GB',
    'X64-30GB',
    'X64-60GB',
    'X64-120GB',
    'DEV1-S',

]
```

**Comment by TC1977** [comment]
Ok nice. It appears this issue may be fixed in [Ansible 2.8](https://docs.ansible.com/ansible/latest/modules/scaleway_compute_module.html?highlight=scaleway).
